tests/functional/codegen/calling_convention/test_default_parameters.py

```diff
@@ -191,17 +192,17 @@ def bar(a: int128, b: int128 = -1) -> (int128, int128):  # noqa: E501
 
     c = get_contract(code)
 
-    assert c.bar(-123) == [-123, -1]
-    assert c.bar(100, 100) == [100, 100]
+    assert c.bar(-123) == (-123, -1)
+    assert c.bar(100, 100) == (100, 100)
 
-    def validate_value(cls, value):
-        pass
+    def visit_int(node, value):
+        return value.to_bytes(32, "big", signed=node.is_signed)
 
-    monkeypatch.setattr("eth_abi.encoding.NumberEncoder.validate_value", validate_value)
```

Comment:
it's pretty confusing what this line is supposed to be doing tbh. if we are trying to bypass the encoder checks, we should just craft the calldata manually